TST: broken tests as data not installed

skipping these tests: https://github.com/pandas-dev/pandas/pull/18873

conda install is finally working. surfacing some failing tests because of data files not being installed.
https://travis-ci.org/pandas-dev/pandas/jobs/319150721
```
/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pandas/tests/io/conftest.py:56: FileNotFoundError

=================================== FAILURES ===================================

 TestCommonIOCapabilities.test_read_fspath_all[reader4-feather-/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pandas/tests/io/data/feather-0_3_1.feather] 

[gw0] linux -- Python 3.5.4 /home/travis/miniconda3/envs/pandas/bin/python


self = <pandas.tests.io.test_common.TestCommonIOCapabilities object at 0x7f93b0cd67b8>

reader = <function read_feather at 0x7f93d86bb268>, module = 'feather'

path = '/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pandas/tests/io/data/feather-0_3_1.feather'


    @pytest.mark.parametrize('reader, module, path', [

        (pd.read_csv, 'os', os.path.join(HERE, 'data', 'iris.csv')),

        (pd.read_table, 'os', os.path.join(HERE, 'data', 'iris.csv')),

        (pd.read_fwf, 'os', os.path.join(HERE, 'data',

                                         'fixed_width_format.txt')),

        (pd.read_excel, 'xlrd', os.path.join(HERE, 'data', 'test1.xlsx')),

        (pd.read_feather, 'feather', os.path.join(HERE, 'data',

                                                  'feather-0_3_1.feather')),

        (pd.read_hdf, 'tables', os.path.join(HERE, 'data', 'legacy_hdf',

                                             'datetimetz_object.h5')),

        (pd.read_stata, 'os', os.path.join(HERE, 'data', 'stata10_115.dta')),

        (pd.read_sas, 'os', os.path.join(HERE, 'sas', 'data',

                                         'test1.sas7bdat')),

        (pd.read_json, 'os', os.path.join(HERE, 'json', 'data',

                                          'tsframe_v012.json')),

        (pd.read_msgpack, 'os', os.path.join(HERE, 'msgpack', 'data',

                                             'frame.mp')),

        (pd.read_pickle, 'os', os.path.join(HERE, 'data',

                                            'categorical_0_14_1.pickle')),

    ])

    def test_read_fspath_all(self, reader, module, path):

        pytest.importorskip(module)

    

        mypath = CustomFSPath(path)

>       result = reader(mypath)


/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pandas/tests/io/test_common.py:152: 

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pandas/io/feather_format.py:112: in read_feather

    return feather.read_dataframe(path, nthreads=nthreads)

/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pyarrow/feather.py:143: in read_feather

    reader = FeatherReader(source)

/home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pyarrow/feather.py:43: in __init__

    self.open(source)

feather.pxi:79: in pyarrow.lib.FeatherReader.open

    ???

io.pxi:622: in pyarrow.lib.get_reader

    ???

io.pxi:411: in pyarrow.lib.memory_map

    ???

io.pxi:390: in pyarrow.lib.MemoryMappedFile._open

    ???

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 


>   ???

E   pyarrow.lib.ArrowIOError: IOError: Failed to open file: /home/travis/miniconda3/envs/pandas/lib/python3.5/site-packages/pandas/tests/io/data/feather-0_3_1.feather


error.pxi:60: ArrowIOError

```

--- a/pandas/tests/io/test_common.py
+++ b/pandas/tests/io/test_common.py
@@ -144,6 +144,7 @@ bar2,12,13,14,15
         with pytest.raises(error_class):
             reader(path)
 
+    @pytest.mark.xfail(reason="not working in 3.5 conda build")
     @pytest.mark.parametrize('reader, module, path', [
         (pd.read_csv, 'os', os.path.join(HERE, 'data', 'iris.csv')),
         (pd.read_table, 'os', os.path.join(HERE, 'data', 'iris.csv')),

